src/Tool/Program.cs

```diff
@@ -35,6 +36,28 @@ public static bool TelemetryOptOut
             => true;
 #endif
 
+        // Creates dictionary of kernelspec file names to the embedded resource path
+        // of all embedded resources found in `Kernel.csproj`.
+        public static Dictionary<string, string> GetEmbeddedKernelResources()
+        {
+            var res = new Dictionary<string, string>();
+            foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
+            {
+                if (String.Equals(assembly.GetName().Name, "Microsoft.Quantum.IQSharp.Kernel"))
+                {
+                    foreach (var resName in assembly.GetManifestResourceNames())
+                    {
+                        // Compute "filename.extension"
```

Comment:
Maybe also worth noting (in the documentation for the function?) that this won't work for EmbeddedResources that have multiple `.`'s in the filename.

Reply:
Would it make more sense then to not only take the last 2 components of the path, but use everything after `Microsoft.Quantum.Kernel.res`? For example, `Microsoft.Quantum.Kernel.res.class.subclass.js` => `class.subclass.js`? Any file that has directories will also be treated in this fashion (which would break imports if any file is trying to reference it).

Reply:
Yes, I think that would also make sense.